frontend/src/actions/transfers/transfer.ts

```diff
@@ -343,10 +347,8 @@ export class Transfer extends MultiStepAction implements Encodable<TransferData>
     );
 
     try {
-      const timestamp = await Promise.race([fulfillmentPromise, expirationPromise]);
-      if (timestamp) {
-        this._requestFulfillment = new RequestFulfillment({ timestamp });
-      }
+      await Promise.race([fulfillmentPromise, expirationPromise]);
+      this._requestFulfillment = new RequestFulfillment({ timestamp: Date.now() });
```

Comment:
Could you chain the `new Request Fulfillment` call to the `fulfillmentPromise` with a `then`? So we only set the fulfillment if the request was really fulfilled. I think this is the safer approach for future changes.

Reply:
Good catch! ✅